marklogic-client-api/src/main/java/com/marklogic/client/impl/DatabaseClientPropertySource.java

```diff
@@ -271,28 +278,34 @@ private DatabaseClientFactory.SSLHostnameVerifier determineHostnameVerifier() {
 	 *                 case the user does not define their own SSLContext or SSL protocol
 	 * @return
 	 */
-	private SSLInputs buildSSLInputs(String authType) {
+	private SSLUtil.SSLInputs buildSSLInputs(String authType) {
 		X509TrustManager userTrustManager = getTrustManager();
 
 		// Approach 1 - user provides an SSLContext object, in which case there's nothing further to check.
 		SSLContext sslContext = getSSLContext();
 		if (sslContext != null) {
-			return new SSLInputs(sslContext, userTrustManager);
+			return new SSLUtil.SSLInputs(sslContext, userTrustManager);
 		}
 
-		// Approaches 2 and 3 - user defines an SSL protocol.
-		// Approach 2 - "default" is a convenience for using the JVM's default SSLContext.
-		// Approach 3 - create a new SSLContext, and initialize it if the user-provided TrustManager is not null.
+		// Approach 2 - user wants two-way SSL via a keystore.
```

Comment:
We had 4 approaches before, we now have 5 - the new approach number 2 is for two-way SSL via a keystore.